spec/fortnox/api/validators/customer_spec.rb

```diff
@@ -1,33 +1,31 @@
 require 'spec_helper'
-require 'fortnox/api/validators/customer'
 require 'fortnox/api/models/customer'
+require 'fortnox/api/validators/context'
+require 'fortnox/api/validators/customer'
 
 describe Fortnox::API::Validator::Customer do
   subject{ described_class.new }
 
-  describe '.validate' do
-    context 'Customer with valid, simple attributes' do
-      let( :customer ){ Fortnox::API::Model::Customer.new( name: 'Test' ) }
-
-      it 'is valid' do
-        expect( subject.validate( customer )).to eql( true )
-      end
-    end
-
-    context 'Customer with invalid sales_account' do
-      let( :customer ){ Fortnox::API::Model::Customer.new( name: 'Test', sales_account: 99999 ) }
+  let( :model_class ){ Fortnox::API::Model::Customer }
```

Comment:
I don't like this tbh. It's obfuscating and only serves the purpose of saving you a search and replace with multiple hits instead of one hit if we ever change the class name. A better option, if you want something shorter, is to do either:
```ruby
using_test_class do
  Customer = Fortnox::API::Model::Customer
end
```
or
```
include Fortnox::API::Model
```
where the last one imports all the constants from `Fortnox::API::Model` into the local scope, so you get access to `Model` in the local scope instead of going through the `Fortnox::API::Model` chain explicitly each time. You might have to wrap this in `using_test_class` as well to ensure no constant overload warnings/errors ...